repair/repair.cc

```diff
@@ -1081,6 +1076,80 @@ future<int> repair_service::do_repair_start(sstring keyspace, std::unordered_map
     auto id = _repair_module->new_repair_uniq_id();
     rlogger.info("repair[{}]: starting user-requested repair for keyspace {}, repair id {}, options {}", id.uuid(), keyspace, id.id, options_map);
 
+    repair_options options(options_map);
+
+    std::vector<sstring> cfs =
+        options.column_families.size() ? options.column_families : list_column_families(db, keyspace);
+    if (cfs.empty()) {
+        rlogger.info("repair[{}]: completed successfully: no tables to repair", id.uuid());
+        co_return id.id;
+    }
+
+    {
+        std::sort(cfs.begin(), cfs.end());
```

Comment:
Why sort?

Reply:
Because I want the order of tables to repair in a more stable way. Btw, we could also sort from the size of the tablet so we can repair small tables first which SM does the same (Not for now). 

```
    using ks_cf_to_uuid_t =
        flat_hash_map<ks_cf_t, table_id, utils::tuple_hash, string_pair_eq>;
    class tables_metadata {
        rwlock _cf_lock;
        std::unordered_map<table_id, lw_shared_ptr<column_family>> _column_families;
        ks_cf_to_uuid_t _ks_cf_to_uuid;
```

Reply:
What does it help with? We need to leave a comment because it's not obvious.

Reply:
For example, when a keyspace have multiple tables, it is better to see repair to finish tables in some order, e.g., ks.standard1, ks.standard2, ks.standar3.  Not a in  random order.